controllers/index.js

```diff
@@ -0,0 +1,11 @@
+// controllers/index.js
+
+(function (controllers) {
+
+    var githubController = require("./githubController");
+    
+    controllers.init = function (app) {
```

Comment:
It's called from server.js (unless I screwed up the merge...)

Sent from my Windows Phone

-----Original Message-----
From: "enomic" <notifications@github.com>
Sent: ‎8/‎18/‎2015 8:17 AM
To: "enomic/enomic" <enomic@noreply.github.com>
Cc: "Zain Rizvi" <zainriz3@gmail.com>
Subject: Re: [enomic] Create organizational structure to the code (viacontrollers) (#23)

In controllers/index.js:
> @@ -0,0 +1,11 @@
> +// controllers/index.js
> +
> +(function (controllers) {
> +
> +    var githubController = require("./githubController");
> +    
> +    controllers.init = function (app) {
What calls this? I don't think the code will work in the current form
—
Reply to this email directly or view it on GitHub.